src/CHIPBackend.hh

```diff
@@ -2060,6 +2054,12 @@ public:
 
   CHIPDevice *PerThreadQueueForDevice = nullptr;
 
+  virtual CHIPEvent *enqueueNativeGraph(CHIPGraphNative *NativeGraph) {
+    return nullptr;
+  }
+  virtual CHIPGraphNative *createNativeGraph() { return nullptr; }
```

Comment:
Do these methods have preconditions? Is it an error to call these if the backends do not override them?